contribute/contribute-pull-requests/add_hook.md

```diff
@@ -0,0 +1,18 @@
+# How to add a new Hook
+Adding a new hook is a process that can be done in only three steps.
+
+## 1) Add a call to Hook::exec()
+
+As you can see on this [PR](https://github.com/PrestaShop/PrestaShop/pull/34431/files), adding a simple Hook::exec() statement is enough to add a new hook. In order for it to be complete, there is a couple more steps to do.
+
+## 2) Modify the hook.xml file
+
+In order to properly register the hook you need to add it to the hook.xml file as seen in this [PR](https://github.com/PrestaShop/PrestaShop/pull/34431/files)
+
+## 3) Add the new hook in the Autoupgrade fixture
+
+Finally, in order to have this new hook available for users upgrading from older versions of PrestaShop, you need to add it to the current version SQL script of the Autoupgrade module, as it can be seen in this [PR](https://github.com/PrestaShop/autoupgrade/pull/672/files).
+
+If you are unsure to which file you should put the code inside autoupgrade module, feel free to ping `@PrestaShop/Committers` 
+
+There is also a [helper](https://github.com/PrestaShop/autoupgrade/pull/577/files) available to add a new hook.
```

Comment:
```suggestion
You can also use a [helper](https://github.com/PrestaShop/autoupgrade/pull/577/files) available to add a new hook during the upgrade process for a given version.
```

Following these three steps will ensure that the hook is correctly:
- added to the database and available to sort modules in it by position
- executed in the code
- added for users upgrading from the old versions